package.json

```diff
@@ -43,7 +43,6 @@
     "!web-test-runner.config.js",
     "!commitlint.config.js",
     "!**/test/**",
-    "!**/testing/**",
```

Comment:
Let's keep excluding all the other testing files and explicitly add the files that we consider to be part of the public API for harnesses that clients can use in their tests:

```
    "!**/test/**",
    "!**/testing/**",
    "testing/harness.{js,js.map,d.ts}",
    "testing/transform-pseudo-classes.{js,js.map,d.ts}",
```

Reply:
I added that :)